sdk/core/azure-core-tracing-opentelemetry/src/main/java/module-info.java

```diff
@@ -7,10 +7,10 @@
     requires io.opentelemetry.api;
     requires io.opentelemetry.context;
 
-    requires transitive io.opentelemetry.semconv;
-
     opens com.azure.core.tracing.opentelemetry to com.fasterxml.jackson.databind;
 
+    opens com.azure.core.tracing.opentelemetry.implementation;
```

Comment:
Does this need to be a general opens or should this be targeted? Right now this is opening the module for reflection to all code

Reply:
Yeah, I added this for reflections in tests. I have seen this before in some of our packages, is the guidance always to keep this targeted?

Reply:
If this is only needed for testing I would use a Maven Surefire configuration such as this: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/search/azure-search-documents/pom.xml#L114